src/components/Sidebar/index.jsx

```diff
@@ -61,25 +69,35 @@ export default class Sidebar extends Component {
 
     // Add Swift directly as there is no `lang=swift` in bugzilla whiteboards
     return (
-      <List disablePadding>
-        {['Swift', ...Object.keys(BUGZILLA_LANGUAGES)].map(language => (
-          <ListItem
-            className={classNames({
-              [classes.active]: language.toLowerCase() === activeLanguage,
-            })}
-            button
-            onClick={this.props.onLanguageClick}
-            id={language}
-            key={language}
-            component={Link}
-            to={`/languages/${language.toLowerCase()}`}>
-            <ListItemIcon>{icons[language] || <WebIcon />}</ListItemIcon>
-            <ListItemText disableTypography className={classes.text}>
-              {language}
-            </ListItemText>
-          </ListItem>
-        ))}
-      </List>
+      <React.Fragment>
```

Comment:
Instead of a fragment, you can use a `<div>` with `display: 'flex'` and then probably use the `space-between` style to add the space between the list and the links. This would avoid having to use `position: absolute`. If you're not familiar with flexbox, https://css-tricks.com/snippets/css/a-guide-to-flexbox/ was a great learning resource for me.

Reply:
for space-between what should I use align-content or justify-content?
@helfi92  
Thanks!

Reply:
I think it should be `justify-content` because it is along the main axis (vertically, you want to spread out the children).